docs/RELEASE-NOTES.rst

```diff
@@ -5,17 +5,27 @@ next-release
 ------------
 Added Functionality
 `````````````````````
-* Support Path Based Routing for Openshift reencrypt routes.
-* OpenShift 4.1, Kubernetes 1.14 support and other dependency package upgrade.
-* Update OpenShift 3.11 GUI with Route admitted status based on BIG-IP response.
+* TBA 
 
 Bug Fixes
 `````````
-* Controller configured to manage-routes now exist with readable help message in logs when router-vserver-addr is not configured.
+* TBA
```

Comment:
Please remove these entries. A single blank line should be kept under `next-release`

Reply:
Ok Thanks

Reply:
The requested changes are addressed.